Detections/Box/BoxUserLoginAsAdmin.yaml

```diff
@@ -0,0 +1,36 @@
+id: b2197d7f-4731-483c-89de-d48606b872da
+name: Box - User logged in as admin
+description: |
+  'Detects when user logged in as admin.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+query: |
+  let lbperiod_start = 60d;

```

Comment:
60d is not recommended, 14d max recommended.

Reply:
updated

Reply:
does not look like this was updated

Reply:
fixed